packages/nft-analytics/src/api/index.ts

```diff
@@ -2,14 +2,17 @@ import { AdapterError, util } from '@chainlink/ea-bootstrap'
 import { Config } from '@chainlink/types'
 
 import { Provider } from './constants'
+import nftperp from './nftperp'
 import rarify from './rarify'
 
 export default (config: Config) => {
-  const provider = util.getEnv('API_PROVIDER')
+  const provider = util.getRequiredEnv('API_PROVIDER')
 
   switch (provider) {
     case Provider.RARIFY:
       return rarify(config)
+    case Provider.NFTPERP:
+      return nftperp(config)
     default: {
       const message = `Unsupported provider: ${provider}. Check API_PROVIDER env var`
       throw new AdapterError({
```

Comment:
This `statusCode` should be 500. Please amend all the adapter ones (including Rarify ones)